plugins/logs/plugin.go

```diff
@@ -891,7 +891,7 @@ func uploadChunk(ctx context.Context, client rest.Client, uploadPath string, dat
 		Do(ctx, "POST", uploadPath)
 
 	if err != nil {
-		return errors.Wrap(err, "Log upload failed")
+		return errors.Wrap(err, "log upload failed")
```

Comment:
[nit] since you're touching the code
```suggestion
		return fmt.Errorf("log upload failed: %w", err)
```

Reply:
Fixed, along with the other occurences of `errors.Wrap` in the same file.